sirepo/server.py

```diff
@@ -474,7 +474,7 @@ def _redirect(req):
         except sirepo.util.SPathNotFound:
             return _not_found(req)
 
-    @sirepo.quest.Spec("require_user", search="SearchSpec")
```

Comment:
This would be insufficient in any event if they bookmarked a simulation. This security has to be pervasive.